core/src/main/java/com/rudderstack/android/sdk/core/RudderOption.java

```diff
@@ -54,8 +56,33 @@ public RudderOption putExternalId(String type, String id) {
         return this;
     }
 
+    public RudderOption putIntegration(@NonNull  String type, @NonNull  boolean enabled)
+    {
+        if(this.integrations == null)
```

Comment:
instead of this can we initialise in the constructor itself?

Reply:
yeah did it